pkg/kudoctl/cmd/test.go

```diff
@@ -46,6 +46,7 @@ func newTestCmd() *cobra.Command {
 	startKUDO := false
 	skipDelete := false
 	skipClusterDelete := false
+	parallel := 0
```

Comment:
why don't we have 8 here since it's the default? 🤔

Reply:
Well we only set `options.Parallel` to `parallel` if it is explicitly set by the user (and the only reason to set it on line 182 is for the help text). Maybe the best thing for me to do is add a comment that the default is enforced in the `RunTests` method.